questionnaire/questionnaire-service.js

```diff
@@ -209,20 +209,18 @@ function createQuestionnaireService({
         const caseReferenceNumber = await retrieveCaseReferenceNumber(questionnaireId);
         const submitted = !!caseReferenceNumber;
 
-        const response = {
-            data: {
-                id: questionnaireId,
-                type: 'submissions',
-                attributes: {
-                    questionnaireId,
-                    submitted,
-                    status,
-                    caseReferenceNumber
-                }
+        const resource = {
```

Comment:
Does the top level `data` attribute need to move?

Reply:
this pattern conforms to the rest of the resources return from their respective functions. I want to return the resource and then shape the request (with the data attribute) later